src/map/handler/Map.Drag.js

```diff
@@ -77,10 +79,6 @@ L.Map.Drag = L.Handler.extend({
 		    .fire('movestart')
 		    .fire('dragstart');
 
-		if (map.options.inertia) {
-			this._positions = [];
-			this._times = [];
```

Comment:
Why remove the initialization in `_onDragStart`? If inertia is enabled, we want to forget the speed of the previous drag(s) whenever a new drag is started; so we want to keep this bit of code.